_posts/2016-11-09-foreman-community-newsletter-january-2016.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: Foreman Community Newsletter - January 2016
+author: Adam Ruzicka
+tags:
+- foreman
+- katello
+- newsletter
+excerpt: |
+  Foreman 1.10 and Katello 2.4 released
```

Comment:
This is never going to make people click through and read the rest of the article :) Add "Read on for details of new plugin releases, upcoming events, and new recordings to watch!"